include/PandoraMonitoring.h

```diff
@@ -406,12 +434,33 @@ class PandoraMonitoring
     const pandora::Pandora *const   m_pPandora;                 ///< The associated pandora instance
     TApplication                   *m_pApplication;             ///< The root application
     TEveManager                    *m_pEveManager;              ///< The eve manager
+    TEveManager                    *m_pEveManager2D;            ///< The eve manager for 2D hits
     TTreeWrapper                   *m_pTreeWrapper;             ///< wrapper around TTree functionality
 
+    TEveViewer                     *m_p3DView;                  ///< Viewer for the 3D view
+    TEveViewer                     *m_p2DUView;                 ///< Viewer for the U 2D view
+    TEveViewer                     *m_p2DVView;                 ///< Viewer for the V 2D view
+    TEveViewer                     *m_p2DWView;                 ///< Viewer for the W 2D view
+
+    TEveScene                      *m_p3DEventScene;            ///< Scene containing the 3D event
+    TEveScene                      *m_p2DUEventScene;           ///< Scene containing the U 2D event
+    TEveScene                      *m_p2DVEventScene;           ///< Scene containing the V 2D event
+    TEveScene                      *m_p2DWEventScene;           ///< Scene containing the W 2D event
+
+    TEveScene                      *m_p3DGeometryScene;         ///< Scene containing the 3D geometry
+    TEveScene                      *m_p2DUGeometryScene;        ///< Scene containing the U 2D geometry
+    TEveScene                      *m_p2DVGeometryScene;        ///< Scene containing the V 2D geometry
+    TEveScene                      *m_p2DWGeometryScene;        ///< Scene containing the W 2D geometry
+
     float                           m_scalingFactor;            ///< TEve works with [cm], Pandora works with [mm]
     bool                            m_openEveEvent;             ///< is set if an Event is open to store objects (hits, clusters,...) in it.
     int                             m_eventDisplayCounter;      ///< counter for the event displays
-
+    double                          m_minXLArTPC;               ///< Minimum x position to which any LAr TPC in the event extends
+    double                          m_maxXLArTPC;               ///< Maximum x position to which any LAr TPC in the event extends
+    double                          m_minYLArTPC;               ///< Minimum y position to which any LAr TPC in the event extends
+    double                          m_maxYLArTPC;               ///< Maximum y position to which any LAr TPC in the event extends
+    double                          m_minZLArTPC;               ///< Minimum z position to which any LAr TPC in the event extends
+    double                          m_maxZLArTPC;               ///< Maximum z position to which any LAr TPC in the event extends
```

Comment:
Could create e.g. a LArTPC extent object to wrap this up, but don't worry about this right now.

Reply:
This would be a good idea for the future.  I will add this to the to do list.